buildkite/scripts/generate-diff.sh

```diff
@@ -5,13 +5,19 @@ BASE=${BUILDKITE_PULL_REQUEST_BASE_BRANCH:-origin/develop}
 
 # Finds the greatest common commit that is shared between these two branches
 # Or nothing if there isn't one
-COMMIT=$(diff -u <(git rev-list --first-parent HEAD) \
-        <(git rev-list --first-parent $BASE) | \
-        sed -ne 's/^ //p' | head -1)
+# COMMIT=$(diff -u <(git rev-list --first-parent HEAD) \
+#        <(git rev-list --first-parent $BASE) | \
+#        sed -ne 's/^ //p' | head -1)
 
-if [[ $COMMIT != "" ]]; then
-  # Get the files that have changed since that shared commit
-  git diff $COMMIT --name-only
+BASECOMMIT=$(git log $BASE -1 --pretty=format:%H) # Finds the commit hash of HEAD of $BASE branch
```

Comment:
More than 1 way to skin a cat for sure though kind of feels like we're dancing around using something like:  `git merge-base` which is generally intended for finding common ancestors.

Reply:
yeah, i'm not trying to find common ancestors i'm just trying to diff all of the files against the latest known commit on the base branch. Git log is just getting me the 1st commit (-1) in the format %H (full length sha hash).